File: Armstrong Number/Armstrong Number.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Tue Oct  6 18:31:36 2020

@author: ravi
"""

n = input("Please enter number ==> ")

sumOfDigit = 0

for i in n:
    i = int(i)
    sumOfDigit += i**3 
if sumOfDigit == int(n):
    print("Armstrong Number")
else:
    print("Not a Armstrong Numbr")

File: Armstrong Number/Armstrong Number.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    int n;
    cout<<"Please Enter Number ==> ";
    cin>>n;
    int temp = n;
    int sumOFDigit = 0;

    while(n>0){
        sumOFDigit += pow(n%10,3);
        n /= 10;
    }

    if(temp==sumOFDigit){
        cout<<"Yes, Given Number is Armstrong Number";
    }else{
        cout<<"Given Number is not Armstrong Number";
    }

    return 0;
}